### DISCUSSION TITLE
Seeing available peers on self-hosted

### DISCUSSION THREAD

**Discussion by Blake-Ballew** [netbirdio/netbird#1298] [Q&A / Support]
I run a self-hosted instance of NetBird. On my user account (with user permissions), I can see all the devices that were added by that account. I've added a server to the network with a setup key, and have configured the user account to be able to access that server and have verified that it can be pinged. Because this server was added via setup key and not the user account, it cannot be seen on the dashboard, so I can only ping it because I know the IP address from viewing the admin account. How is a user account supposed to know what peers are available to connect to if the user can only see their machines on the dashboard?

**Comment by Darin755** [comment]
Maybe I'm missing something but shouldn't devices added with a key show up automaticly? Maybe try deleting the device and readding.
